Civi.php

```diff
@@ -95,6 +95,7 @@ public static function service($id) {
   public static function reset() {
     self::$statics = array();
     Civi\Core\Container::singleton();
+    \Civi::service('settings_manager')->useDefaults();
```

Comment:
Hmm, regarding `CiviTestListener`, it's interesting that `CRM_Core_Config::singleton(TRUE,TRUE)` is called 3 lines higher...